influxd.yaml

```
@@ -61,6 +61,54 @@ subpackages:
         - bash
         - yq
 
+  - name: ${{package.name}}-${{package.version}}-bitnami-compat
+    description: "compat package with bitnami/influxdb image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: influxdb
+          version-path: 2/debian-12
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxd/bin
+          cp ${{targets.destdir}}/usr/bin/influxd ${{targets.subpkgdir}}/opt/bitnami/influxd/bin/influxd
+    test:
+      environment:
+        contents:
+          packages:
+            - curl
+      pipeline:
+        - name: version tests
+          runs: |
+            /opt/bitnami/influxd/bin/influxd version | grep ${{package.version}}
+        - name: Start the influxd server
+          uses: test/daemon-check-output
+          with:
+            start: |
+              /opt/bitnami/influxd/bin/influxd run
```

let's start with this given this is the entrypoint of the image and then we'll catch some issues that may come at image level.
this will also be robust testing in here. 
```suggestion
              /opt/bitnami/scripts/influxdb/entrypoint.sh /opt/bitnami/scripts/influxdb/run.sh
```